src/modules/networking/tests/NetworkingTests.cpp

```diff
@@ -345,6 +502,32 @@ namespace OSConfig::Platform::Tests
 
         EXPECT_NE(payload, nullptr);
         delete payload;
+
+        testModule.runCommandCount = 0;
+        testModule.returnValues[4] = testCommandOutputGlobalDnsServers;
+
+        result = testModule.Get(nullptr, nullptr, nullptr, &payload, &payloadSizeBytes);
+
+        EXPECT_EQ(result, MMI_OK);
+
+        resultString = std::string(payload, payloadSizeBytes);
+        EXPECT_STREQ(resultString.c_str(), payloadExpectedGlobalDnsServers);
+
+        EXPECT_NE(payload, nullptr);
+        delete payload;
+
+        testModule.runCommandCount = 0;
+        testModule.returnValues[4] = testCommandOutputOnlyGlobalDnsServers;
+
+        result = testModule.Get(nullptr, nullptr, nullptr, &payload, &payloadSizeBytes);
+
+        EXPECT_EQ(result, MMI_OK);
```

Comment:
This is the wrong check. That testModule.Get must fail with a null session handle, not return MMI_OK.

Bug in test and also bug in module.